pints/_mcmc/__init__.py

```diff
@@ -377,7 +383,19 @@ def __init__(self, log_pdf, chains, x0, sigma0=None, method=None):
         self._max_iterations = None
         self.set_max_iterations()
 
-        # TODO: Add more stopping criteria
+    def log_density_storage(self):
```

Comment:
@ben18785 and @martinjrobins what do we think of using the term "density" here? It sounds good to me, but we've stuck with "LogPDF" in most of the remainder!

Reply:
@MichaelClerx We use LogPDF when creating a function (as it is a log probability density function) whereas a density is a value (outputted by such a function) that we want to record. So reckon keep as is...

Happy to merge this now?

Reply:
Well no, we're still using `log_pdf` pretty much everywhere else, including in this class e.g. `set_log_pdf_filename` which does the same thing but on disk... I'm also wondering about the nested and smc methods where we have multiple densities (and multiple LogPDFs for that matter) but we have input arguments `log_pdf` and `log_prior` so it would make sense to have names like `save_log_pdf_evaluations` and `save_log_prior_evaluations` rather than `save_density` which could mean either